### PR TITLE
Changing backup mechanism from file copy to per-sheet copy

### PR DESCRIPTION
**Pull request by susanashlock**
Previously, the backup script copied the spreadsheet file, which also made copies of the associated forms and scripts).  This is now updated to create of a new Spreadsheet, and then copy sheets from original to backup one at a time.

### CODE REVIEW

File: public/scripts/backupSheets.js [hunk 1]
```diff
@@ -49,39 +43,57 @@ function backupSheets() {
...
-  for (let i = 0; i < sheets.length; i++) {
-    const name = "Backup of " + sheets[i].name + " " + formattedDate;
-  
-    const file = DriveApp.getFileById(sheets[i].sheet_id);
-  
-    const destination = DriveApp.getFolderById(sheets[i].backup_folder_id);
-  
-    // Makes a copy of file with name "name" in the "destination" folder.
-    file.makeCopy(name, destination);
+  for (let i = 0; i < SHEETS.length; i++) {
```
Comment (awong-dev) [inline public/scripts/backupSheets.js:59]: nit;

for (const sheet of SHEETS) {
  const name = "Backup of " + sheet.name + " " + formattedDate;
  ....
}

File: public/scripts/backupSheets.js [hunk 2]
```diff
@@ -49,39 +43,57 @@ function backupSheets() {
...
-  cleanupFormCopies(sheetsFolder);
 }
 
-function cleanupFormCopies(folder) {
-  const filesIterator = folder.getFiles();
+/*
+  If we do a simple file copy, we make a copy of the spreadsheet
+  and also the associated form(s) and script files, which is bad.
+  To work around this, we create a new spreadsheet and copy in the tabs
+  (sheets) from the source sheet.
```
Comment (awong-dev) [inline public/scripts/backupSheets.js:72]: O.O

Whoa. how in the world did you even think about  doing this. 👏

Comment (susanashlock) [reply public/scripts/backupSheets.js:72]: Stack overflow, of course.  Should I cite my source in a comment here?

File: public/scripts/backupSheets.js (hunk 1 above)
Comment (susanashlock) [reply public/scripts/backupSheets.js:59]: Ooooooh.  Very cool.  Thank you, sensei. 🙇‍♀

File: public/scripts/backupSheets.js (hunk 2 above)
Comment (susanashlock) [reply public/scripts/backupSheets.js:72]: Credit added.

**Pull request merged**